example.pj

```diff
@@ -1,130 +1 @@
-minimal-version 1.1
```

Comment:
Why remove this file? It's mentioned in the readme so if you remove it you should also remove that mention.

Reply:
I didn't remove it, I just left only one line(command), but makes sense to
include one 1 example of each command.